pkgs/development/tools/vcpkg-tool/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cacert
+, cmake
+, cmakerc
+, curl
+, fmt
+, git
+, gzip
+, makeWrapper
+, meson
+, ninja
+, openssh
+, python3
+, zip
+, zstd
+}:
+stdenv.mkDerivation {
+  pname = "vcpkg-tool";
+  version = "2023-08-09";
+
+  src = fetchFromGitHub {
+    owner = "microsoft";
+    repo = "vcpkg-tool";
+    rev = "2023-08-09";
+    hash = "sha256-Prt49KTpYcgL8tdDQlCL0pDFQq+YxwsZ0NqFN6NKG+E=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    cmakerc
+    fmt
+    ninja
+    makeWrapper
+  ];
+
+  patches = [ change-lock-location.patch ];
+
+  cmakeFlags = [
+    "-DVCPKG_DEPENDENCY_EXTERNAL_FMT=ON"
+    "-DVCPKG_DEPENDENCY_CMAKERC=ON"
+  ];
+
+  postFixup = let
```

Comment:
Fix the indentation.

(Sorry, it is harder to write code ~~outside Emacs~~ in the GitHub web interface.)